shared/js/fb/fb_data_reader.js

```diff
@@ -4,8 +4,8 @@
 // To use this library you need to include 'shared/js/fb/fb_request.js'
 // WARNING: This file lazy loads 'shared/js/fb/fb_tel_index.js'
 
-var fb = this.fb || {};
```

Comment:
This is what I found:
```
Searching 12154 files for "this.fb" (whole word)

/home/***/gaia/shared/js/contacts/import/facebook/fb_contact_utils.js:
    4  
    5  /* jshint ignore:start */
    6: var fb = this.fb || {};
    7  /* jshint ignore:end */
    8  

/home/***/gaia/shared/js/contacts/import/facebook/fb_query.js:
    4  
    5  /* jshint ignore:start */
    6: var fb = this.fb || {};
    7  
    8  // In an only worker execution context this would not be necessary
    9  var self = this;
   10: fb.utils = this.fb.utils || {};
   11  /* jshint ignore:end */
   12  

3 matches across 2 files
```